<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageCoursesInterfaceWindow</class>
 <widget class="QWidget" name="ManageCoursesInterfaceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="termSelectOption">
   <property name="geometry">
    <rect>
     <x>247</x>
     <y>10</y>
     <width>151</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="termLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>30</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Term</string>
   </property>
  </widget>
  <widget class="QListView" name="courseListView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteCourseOption">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>88</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="createCourseOption">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Course</string>
   </property>
  </widget>
  <widget class="QPushButton" name="editCourseOption">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>171</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify Course Basic Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="courseLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Courses</string>
   </property>
  </widget>
  <widget class="QListView" name="registeredStudentListView">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QListView" name="assignedTextbookListView">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="studentLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Registered Students</string>
   </property>
  </widget>
  <widget class="QLabel" name="TextbookLabel">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Assigned Textbooks</string>
   </property>
  </widget>
  <widget class="QPushButton" name="enrollStudentOption">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>120</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Enroll Student</string>
   </property>
  </widget>
  <widget class="QPushButton" name="linkTextbookOption">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>210</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Link Textbook</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
